ex1/exceptions.py

```diff
@@ -37,7 +37,33 @@
 # Use the function search_n from the functions module inside a new function
 # also named search_n. The function should do the same as functions.search_n
 # but if the variable is not found in the list then raise a ValueError.
-
+def search_n(things, item):
+    """
+    proofes, if a variable can be found within a given list of numbers and strings
+    - if not it raises a ValueError
+    - else it returns the tuple of the index where the variable was found  and
+    the variable itself
+    Parameters
+    ----------
+    things: list
+        list to search in
+    item: integer, float_number or string
+    Returns
+    ----------
+    found: a_tuple
+        (index, variable)
+    """
+    i = 0
+    while i < len(things):
```

Comment:
There is no need for a loop here since the `in` comparison you used in the `search_n` function does the job. But if you use a loop you can take advantage of `enumerate` since it takes care of increasing the counter.